src/main/java/com/booksaw/betterTeams/events/MCTeamManagement.java

```diff
@@ -119,7 +127,13 @@ public void remove(Player player) {
 
 	@EventHandler
 	public void playerJoinEvent(PlayerJoinEvent e) {
-		Bukkit.getScheduler().runTaskAsynchronously(Main.plugin, () -> displayBelowName(e.getPlayer()));
+		final Player player = e.getPlayer();
```

Comment:
Why in this case the elaboration? And in others you just switch the runner for the included library?